pkg/model/components/containerd.go

```diff
@@ -49,19 +49,19 @@ func (b *ContainerdOptionsBuilder) BuildOptions(o interface{}) error {
 		case b.IsKubernetesGTE("1.26") && b.IsKubernetesLT("1.26.5"):
 			fallthrough
 		case b.IsKubernetesGTE("1.27") && b.IsKubernetesLT("1.27.2"):
-			containerd.Version = fi.PtrTo("1.6.20")
+			containerd.Version = fi.PtrTo("1.7.13")
 			containerd.Runc = &kops.Runc{
-				Version: fi.PtrTo("1.1.5"),
+				Version: fi.PtrTo("1.1.12"),
```

Comment:
@hakman at it seems that you updated last time those other containerd + runc versions was there some reason that all versions were not updated? I am just thinking could we update all versions to use runc 1.1.12 & containerd 1.7.13?

Reply:
Yes, runc must match that version of containerd and kubelet, otherwise it will not work.
Please leave as is. People should update K8s version if they want "better" support.